templates/views/reservation_details.twig

```diff
@@ -52,7 +50,8 @@
             </div>
           </div>
         </div>
-
+        <a hrf="#" data-id="{{ event_key }}" id="get-event-key"></a>
```

Comment:
`hrf` should be `href`. Alternatively you could use a `<button type="button"></button>` element since these aren't true links.